tests/construction-signing/main.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+import (
+	"context"
+	"crypto/sha512"
+	"encoding/json"
+	"fmt"
```

Comment:
Please split imports into groups (runtime library, external, internal).

Reply:
ok

which ones do you mean by "runtime library"? the built in modules?

Reply:
Yeah.